2020-05-03 21:13:00,214 DEBUG    root            Loaded Command Group: ['gcloud', 'beta', 'compute']
2020-05-03 21:13:00,272 DEBUG    root            Loaded Command Group: ['gcloud', 'beta', 'compute', 'ssh']
2020-05-03 21:13:00,275 DEBUG    root            Running [gcloud.beta.compute.ssh] with arguments: [--project: "celtic-vent-271705", --zone: "asia-southeast1-b", [USER@]INSTANCE: "instance-personal"]
2020-05-03 21:13:02,833 WARNING  root            Could not open the configuration file: [/home/bram/.config/gcloud/configurations/config_default].
2020-05-03 21:13:02,834 DEBUG    root            (gcloud.beta.compute.ssh) You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials, or if you have already logged in with a
different account:

  $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
Traceback (most recent call last):
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 983, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 807, in Run
    resources = command_instance.Run(args)
  File "/opt/google-cloud-sdk/lib/surface/compute/ssh.py", line 154, in Run
    client = holder.client
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/base_classes.py", line 67, in client
    self._client = client_adapter.ClientAdapter(
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/api_lib/compute/client_adapter.py", line 55, in __init__
    self._client = client or core_apis.GetClientInstance(
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/api_lib/util/apis.py", line 331, in GetClientInstance
    return apis_internal._GetClientInstance(api_name, api_version, no_http, None,
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/api_lib/util/apis_internal.py", line 169, in _GetClientInstance
    http_client = http_creds.Http(
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/core/credentials/http.py", line 91, in Http
    creds = store.LoadIfEnabled(allow_account_impersonation, use_google_auth)
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 452, in LoadIfEnabled
    return Load(
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 527, in Load
    cred = _Load(account, scopes, prevent_refresh, use_google_auth)
  File "/opt/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 567, in _Load
    raise NoActiveAccountException(
googlecloudsdk.core.credentials.store.NoActiveAccountException: You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials, or if you have already logged in with a
different account:

  $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
2020-05-03 21:13:02,835 ERROR    root            (gcloud.beta.compute.ssh) You do not currently have an active account selected.
Please run:

  $ gcloud auth login

to obtain new credentials, or if you have already logged in with a
different account:

  $ gcloud config set account ACCOUNT

to select an already authenticated account to use.
